src/components/buttonComponent/buttonComponent.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
```

Comment:
```suggestion
import React from "react";
import "./buttonStyles.css";

interface Props {
  children?: React.ReactNode;
  onClick: () => void;
  variant?: "homeScreen" | "formScreen" | "signupScreen";
}

const Button: React.FC<Props> = ({
  variant = "homeScreen",
  children,
  onClick,
}) => {
  return (
    <button className={`base ${variant}`} onClick={onClick}>
      {children}
    </button>
  );
};

export default Button;
```

Example of how you can handle different button styles by introducing a simple prop like `variant`.  This way you have 

- more strict typings so you have better control over the component, rather than letting users pass any class name they want
- you avoid re-introducing the `.base` class name multiple times and avoid breaking the DRY principle

Reply:
Do you want me to change it using variant?

Reply:
Implemented!